templates/_resources.tpl

```diff
@@ -0,0 +1,45 @@
+{{/* vim: set filetype=mustache: */}}
+
+{{/*
+Return a resource request/limit object based on a given preset.
+These presets are for basic testing and not meant to be used in production
+{{ include "deploy.resources.preset" (dict "type" "nano") -}}
+*/}}
+{{- define "deploy.resources.preset" -}}
```

Comment:
CC is not needing so many resources, so I would split the presets to the CC, separated from master and worker.